src/bundle/Resources/public/ts/components/Badge.ts

```diff
@@ -0,0 +1,85 @@
+import Base from '../shared/Base';
+
+const THRESHOLD = {
+    medium: 100,
```

Comment:
here https://github.com/ibexa/design-system/pull/19/files#diff-23ad5292c814372aeee247a0c9ec9850784dcf3c440463f1a2c51608ca01325aR8 you used enum for keys, maybe here you could as well? :)